src/storage/FileserverFileLoader.cpp

```diff
@@ -93,24 +139,38 @@ std::vector<uint8_t> FileserverFileLoader::loadFunctionWamrAotFile(
 std::vector<uint8_t> FileserverFileLoader::loadSharedObjectObjectFile(
   const std::string& path)
 {
-    std::string url = faabric::util::getSharedObjectObjectUrl();
+    std::string urlPath = "sobjobj";
```

Comment:
The logic around these hard-coded strings is to match the upload server, and is currently held in Faabric which is odd. Moving it here will allow us to remove more Faasm-specific stuff from Faabric.